MarsQASpecFlowProject/StepDefinitions/LanguageStepDefinitions.cs

```diff
@@ -0,0 +1,93 @@
+using MarsQASpecFlowProject.Pages;
+using MarsQASpecFlowProject.Utilities;
+using OpenQA.Selenium.Chrome;
+
+namespace MarsQASpecFlowProject.StepDefinitions
+{
+    [Binding]
+    public class LanguageStepDefinitions : CommonDriver
+    {
+        LoginPage loginPageObject = new LoginPage();
```

Comment:
please use constructor 
https://www.knowledgehut.com/tutorials/csharp/csharp-constructors

Reply:
please fix this in all steps

Reply:
Added constructors in all the steps.